### PR TITLE
Making Up a Pull Request 

### PR DESCRIPTION
**Pull request by CTUCKER0900**
## Proposed changes

Just giving a pull request a go and seeing what happens.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- X[ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [X ] I have added necessary documentation (if appropriate)
- [ ] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ X] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?

@TRENDS-Taran-Uppal

### CODE REVIEW

File: Callum Text [hunk 1]
```diff
@@ -0,0 +1,17 @@
+This is me staging my text :)
+
+Let's hope this works and isn't as hard as R
+
+Praying for the best 
+
+Does this work?
```
Comment (DfT-Jack-Casey) [inline Callum Text:7]: yes, it does work!

**Review by DfT-Jack-Casey — COMMENTED** [review]
good job Callum

File: Callum Text [hunk 2]
```diff
@@ -0,0 +1,17 @@
+This is me staging my text :)
+
+Let's hope this works and isn't as hard as R
+
```
Comment (TRENDS-Taran-Uppal) [inline Callum Text:4]: I think this training will cover 99% of the main uses of Github, so hopefully it won't be as hard!

**Review by TRENDS-Taran-Uppal — APPROVED** [review]
your text edits have come through well, happy to approve

**Pull request closed without merging**